examples/python_sdk/cancel_create_experiment.py

```diff
@@ -0,0 +1,27 @@
+from determined.common.experimental.determined import Determined
+
+
+def cancel_experiment(det: Determined) -> None:
+    exp1 = det.get_experiment(experiment_id=1)
+    print("Cancelling experimet id: {}".format(exp1.id))
+    exp1.cancel()
+
+
+def create_experiment_wait(det: Determined) -> None:
+    exp2 = det.create_experiment(
+        exp_config="../tutorials/mnist_pytorch/const.yaml",
+        model_dir="../tutorials/mnist_pytorch",
+    )
+    exp2.activate()
+    exp2.wait_till_complete()
+    print("Experiment {} {} is complete".format(exp2.config["description"], exp2.id))
+
+
+def main():
+    det = Determined(master="http://localhost:8080", user="determined")
```

Comment:
In an example that we show users, we should be using canonical names that are convenient for us in all examples, and also convenient for users to follow.  This will help us build a good ecosystem because when third-party blog posts get written, they can follow the same canonical names and we'll all be on the same page.

In this case, `det` is the canonical name for the determined library, a la `import determined as det`.  So I'd propose that we standardize to `D = Determined()`.

Reply:
updated to use D.